Makefile

```diff
@@ -88,6 +88,7 @@ build-after-poetry:  # continuation of build after poetry install
 	make npm-setup-if-needed
 	poetry run python setup_eb.py develop
 	make fix-dist-info
+	poetry run prepare-local-dev
```

Comment:
You might want to ensure inserts are loaded when this is run

Reply:
I'm not sure I understand why inserts are relevant to this. This is just about setting up details that need to be user-specific, like the indexing prefix. Can you elaborate on your concern?

Yes, this is a very old PR. I was surprised it was never merged, but it fortunately seems not to have gotten stale, so I'm trying to revive it.